deployments/simple-deploys.ps1

```diff
@@ -0,0 +1,29 @@
+    
+    #shows how to import modules
+    Import-Module .\azure\resources\pf-key-vault\pf-key-vault.psd1
+    Import-Module .\azure\resources\pf-log-analytics\pf-log-analytics.psd1
+
+    #The purpose of this script is to show you can still work very simply with powerform and not have to get to overly complex
+
+    ##############################################################################################################
+    #
+    #              Get a resource
+    #
+    #############################################################################################################
+    #Shows how to get a resource.  No need to set a deployment context, build context, etc.
+
+    $kv = Get-PfKeyVault -SubscriptionName "MYOCW-KKZH-D" -ResourceGroupName "RG-MYOCW-KKZH-ADM-D" -Name "KV-MYOCW-KKZH-ADM-C1-D01"
```

Comment:
Wanted to put together a "simple" deploy scenario now.  I don't want it to get lost with all the complex stuff now, that at the end of the day you can still get or deploy a single resource pretty seamlessly